R/get_functions.R

```diff
@@ -83,7 +83,20 @@ get_function_internal <- function(design, data = NULL, start, end, pred, results
   
   design_subset <- Filter(pred, design[start:end])
   
-  run_design_internal.design(design_subset, current_df = data, results = results)[[what]]
+  ret <- run_design_internal.design(design_subset, current_df = data, results = results)[[what]]
   
-}
+  if(what == "estimates_df" && class(ret$estimator_label) != "character"){
```

Comment:
Labels are optional, so don't print the warning if null.

Reply:
ah, thanks

Reply:
Also character isn't really a class, its either a type or mode, I forget which.

Reply:
ah yes thanks will fix